clusters/app.ci/prow/03_deployment/publicize.yaml

```diff
@@ -41,6 +41,8 @@ items:
           args:
           - --dry-run=false
           - --endpoint=/
+          - --github-endpoint=http://ghproxy
```

Comment:
Looks like we generally include a fallback direct endpoint (`- --github-endpoint=https://api.github.com`) in our other deployments:

https://github.com/openshift/release/blob/38f9e244a0d10220958cbd4065304e31b9ba9db4/clusters/app.ci/prow/03_deployment/hook.yaml#L73-L74

https://github.com/openshift/release/blob/38f9e244a0d10220958cbd4065304e31b9ba9db4/clusters/app.ci/prow/03_deployment/plank.yaml#L56-L57

Reply:
addressed. Thanks